pull-request/src/person.js

```diff
@@ -11,5 +11,7 @@ Person.prototype.fullname = function() {
 }
 
 Person.prototype.wishHappyBirthday = function() {
+
     return "Happy birthday " + this.firstName;
+    this.age++;
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Remove this code after the "return" statement. ([javascript:UnreachableCode](http://localhost:9750/coding_rules#rule_key=javascript%3AUnreachableCode))

[See it in SonarQube](http://localhost:9750/project/issues?id=training%3Apull-request&pullRequest=48&issues=AWg8syC16PH_c9BW3X-U&open=AWg8syC16PH_c9BW3X-U)

[ISSUE_KEY]: AWg8syC16PH_c9BW3X-U